client/dist/index.css

```diff
@@ -3,15 +3,15 @@
 },
 
 @font-face {
-  font-family: 'Cereal';
+  font-family: 'Cereal Book';
   src: url('./assets/fonts/AirBnB-Cereal/AirbnbCerealApp-Book.woff2') format('woff2'),
        url('./assets/fonts/AirBnB-Cereal/AirbnbCerealApp-Book.woff') format('woff');
   font-weight: normal;
   font-style: normal;
 }
 
 @font-face {
-  font-family: 'Cereal';
+  font-family: 'Cereal Light';
```

Comment:
Here (lines 14-36 and in some other places) you have the font name as just 'Cereal' and you define the font names of some others, like 'Cereal Light' and 'Cereal Extra Bold'. After talking to anthony - we should call all font faces 'Cereal' and just rely on the font-weights to determine which one gets used. 

For example - on line 63 you use 'Cereal Light'. After you make the above changes, you can just use 'Cereal' and set the font-weight to 300. I think it still works the same, but this is the best-practice protocol according to Anthony.